### PR TITLE
Module cleanup (Macros)

### PR DESCRIPTION
**Pull request by Winter259**
More work for #88
I'll make another PR for safemode's indentation.

### CODE REVIEW

File: addons/weaponselect/functions/fnc_findNextGrenadeMagazine.sqf [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 private ["_type", "_allMags", "_allMuzzles", "_magazines"];
```
Comment (PabstMirror) [inline addons/weaponselect/functions/fnc_findNextGrenadeMagazine.sqf]: looks good, don't need this `_type` private as it's in PARAMS

Comment (Winter259) [inline addons/weaponselect/functions/fnc_findNextGrenadeMagazine.sqf]: Thanks for catching that :+1:

File: addons/weaponselect/functions/fnc_throwGrenade.sqf [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 #include "script_component.hpp"
 
-private ["_unit", "_weapon", "_magazine"];
+private ["_magazine"];
+
+PARAMS_2(_unit,_weapon);
 
-_unit = _this select 0;
-_weapon = _this select 1;
 _magazine = _this select 5;
```
Comment (ulteq) [inline addons/weaponselect/functions/fnc_throwGrenade.sqf]: This does not increase readability in my opinion.

**Comment by ulteq** [comment]
The rest is. :+1:

File: addons/weaponselect/functions/fnc_throwGrenade.sqf (hunk 2 above)
Comment (esteldunedain) [inline addons/weaponselect/functions/fnc_throwGrenade.sqf]: I agree, we shouldn't use PARAMS_X or EXPLODE_X_PVT if not all the variables are extracted.

**Pull request merged**